downstream/modules/platform/proc-import-mesh-ca.adoc

```diff
@@ -0,0 +1,31 @@
+[id="proc-import-mesh-ca_{context}"]
+
+= Importing a Certificate Authority (CA) certificate
+
+A Certificate Authority (CA) is used to certify and sign execution nodes in an {AutomationMesh} environment. Users who wish to provide their own CA can do so by specifying the path to the certificate and the private RSA key file in the `inventory` file of their {PlatformName} installer.
+
+NOTE: Users are not required to have a CA prior to their {AutomationMesh} installation. Users without a CA can proceed with their {PlatformNameShort} installation where the installer will generate a CA for the user using OpenSSL.
+
+.Procedure
+
+. Open the `inventory` file for editing.
+. Add the `mesh_ca_keyfile` variable and specify the full path to the CA certificate file (.crt).
+. Add the `mesh_ca_certfile` variable and specify the full path to the private RSA key (.key).
+. Save the changes to the inventory file.
+
+.Example
+[subs="+quotes"]
+----
+[all:vars]
+mesh_ca_keyfile=/tmp/__<mesh_CA>__.key
+mesh_ca_certfile=/tmp/__<mesh_CA>__.crt
+----
+
+With the CA files added to the inventory file, run the installer to apply the CA. This process copies and stores the CA on each control node on your network. By default, the certfile and keyfile will be added to `/etc/receptor/tls/ca/`.
```

Comment:
My mistake, it will copy the CA to each node (control and execution nodes) on the mesh.